### PR TITLE
Updated working model of perceiver tranform

### PR DESCRIPTION
**Pull request by Akshath-K**
The project has been modified, and the model works as intended on the ADNI brain dataset. It does not show 100% accuracy on everything anymore, as the dataset path was changed correctly to work on split datasets, image conversion was appropriately done, and RGB was not used as all images are in grayscale. Image' labelling was fixed to identify the difference between AD and NC. A validation set was also added to work correctly after each training epoch.  The input dimensions of the image the model was being parsed have been changed accordingly to predict correctly. Finally, the plots and visualization have also been updated according to the new working code and results.

### CODE REVIEW

**Comment by LinfengLiu98** [comment]
**This is an initial inspection, no action is required at this point**

Difficulty: Hard

Readme:

- Need more explanation on how Perciever architecture works and how you developed your algorithm.
- No plots of training and val accuracy
- Reference section is missing

Commit messages: 

- clear
- submitted commits on two days.

Code:
- Uses Perceiver
- Commented
- Good model design

Functionality/Performance:
- Test accuracy is **50.44%**
- Loss and accuracy plots are unclear (unclear legends).
- Only have training ACC plot, no val ACC plot 
- No evidence of patient-level split
- Preprocssing data correctly
- No train, val and test split. **Selected model only based on test set results.**
- No hyperparameter tuning.
- The reference section is missing

**Comment by shakes76** [comment]
# Marking
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate design and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem , but poor performance < 0.6 -1
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage, no validation plot -1
Module present
Commenting
No Data leakage, no patient level split -1, no validation set -1
Difficulty: Hard
## Commit Log
Meaningful commit messages
Progressive commits used but over two days -1
## Documentation
ReadMe acceptable/good
Model/technical explanation
Good Description and Comments 
Markdown used and PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
Feedback required remove pyc files and move README into folder restore repo README -2
Request Description minimal -1

**Comment by shakes76** [comment]
Feedback marks possible +2 if the requested changes are made (see above).

**Comment by wangzhaomxy** [comment]
No feedback attempt and no feedback marks granted.
